apps/entry/factories.py

```diff
@@ -3,14 +3,15 @@
 from factory.django import DjangoModelFactory
 
 from gallery.factories import FileFactory
-
+from analysis_framework.factories import AnalysisFrameworkFactory
 from .models import Entry
 
 
 class EntryFactory(DjangoModelFactory):
     entry_type = fuzzy.FuzzyChoice(Entry.TagType)
     excerpt = fuzzy.FuzzyText(length=100)
     image = factory.SubFactory(FileFactory)
+    analysis_framework = factory.SubFactory(AnalysisFrameworkFactory)
```

Comment:
Let's pass this manually. Basically, entries will not be listed in graphql if the AF is different from projects' AF.